ha-hoymiles-wifi-addon/run.sh

```diff
@@ -31,15 +31,26 @@ if [[ -z "$MQTT_BROKER_HOST" ]]; then
   exit 1
 fi
 
-if [[ -z "$MQTT_USERNAME" ]]; then
-  echo "The mqtt_username is not configured."
-  exit 1
+# The host names are mandatory information
+echo "inverter_host = \"$INVERTER_HOST\" \n\
+\n\
+[home_assistent] \n\
+host = \"$MQTT_BROKER_HOST\"\n"
+
+# Write mqtt broker username if specified
+if [[ -n "$MQTT_USERNAME" ]]; then
+  echo "username = \"$MQTT_USERNAME\"\n" >> /usr/local/bin/config.toml
```

Comment:
the application will try to read the `config.toml` from the current working directory, so the output path should be `./config.toml` instead